### PR TITLE
Bug fix/re add ability udf emit error

### PR DESCRIPTION
**Pull request by dothebart**
forward port of https://github.com/arangodb/arangodb/pull/11995

### CODE REVIEW

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/10696/

**Comment by dothebart** [comment]
blue except for unrelated errors.

File: arangod/V8Server/v8-vocbase.cpp [hunk 1]
```diff
@@ -519,6 +520,42 @@ static void JS_ParseAql(v8::FunctionCallbackInfo<v8::Value> const& args) {
   TRI_V8_TRY_CATCH_END
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief registers a warning for the currently running AQL query
+/// this function is called from aql.js
```
Comment (jsteemann) [inline arangod/V8Server/v8-vocbase.cpp:525]: The comment is not true, so let's remove it.
```suggestion
```

**Comment by jsteemann** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/10717/

**Comment by jsteemann** [comment]
Tests blue

**Pull request merged**